content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -132,7 +132,10 @@ Modifying stage 'evaluate' in 'dvc.yaml'
 
 ```
 
-Now let's view the plots:
+Now let's view the plots. You can run `dvc plots show` on you Terminal (shown
```

Comment:
```suggestion
Now let's view the plots. You can run `dvc plots show` on your Terminal (shown
```

Minor question out of curiosity: why capitalize Terminal?

Reply:
Good Q. Not sure why it was capitalized here, changed.